daniellamarr/fastfoodfast-reactjs
# Chore(Setup Eslint):

#### What does this PR do?
- This PR sets up eslint
#### Description of Task to be completed?
- Add eslint rules to the codebase
- Use airbnb style guide
- Refactor codebase to adhere to eslint airbnb style guide
#### How should this be manually tested?
- Pull this repository off this branch
- Run `npm install` to install packages
- Then make a deliberate bad coding practice to see eslint flag errors
#### Any background context you want to provide?
- Make sure to install eslint for your IDE
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164043258
#### Screenshots (if appropriate)
- N/A
#### Questions:
- N/A
